packages/showcase/src/user-controls/code-snippet/code-snippet.scss

```diff
@@ -11,3 +13,35 @@ code-snippet {
   display: grid;
   grid-template-rows: max-content max-content;
 }
+
+.note {
+  // highlighted article description
+  &::before {
+    content: "note:";
+    display: block;
+
+    font-size: var(--mer-font__size--sm);
+    font-weight: var(--mer-font__weight--semi-bold);
+
+    margin-block-end: var(--mer-spacing--2xs);
+  }
+
+  display: flex;
+  flex-direction: column;
+  gap: var(--mer-spacing--xs);
+  max-inline-size: $articleMaxInlineSize;
+
+  color: var(--mer-color__neutral-gray--400);
+  font-size: var(--mer-font__size--2xs);
+
+  background-color: var(--mer-surface__elevation--01);
+  border-inline-start: 2px solid var(--mer-border-color__on-surface);
+  border-radius: var(--mer-border__radius--sm);
+
+  padding-block: var(--mer-spacing--sm);
+  padding-inline: var(--mer-spacing--md);
+}
+
+.c-green {
+  color: #30cbb8;
```

Comment:
Why doesn't this color use a token? We should not use colors that the DS does not provide.

Reply:
@ncamera Because the green tokens look dull on text over the _code_ background color.
On the following screenshot, over the left `--mer-color__message-green--100`, and over the right the custom fixed `#30cbb8;`. 

![Captura de pantalla 2025-01-25 a las 3 25 37 a  m](https://github.com/user-attachments/assets/cedbc7fd-9d24-4e43-a275-a4997245ec19)